bundles/org.openhab.binding.grundfosalphareader/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="grundfos_flow">
+		<item-type>Number:VolumetricFlowRate</item-type>
+		<label>Current Flow</label>
+		<description>Current flow</description>
+		<category>flow</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="grundfos_head">
+		<item-type>Number:Length</item-type>
+		<label>Current Head</label>
+		<description>Current Head</description>
+		<category>water</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="grundfos_temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Current Pump Temperature</label>
+		<description>Current Pump Temperature</description>
+		<category>temperature</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<thing-type id="mi401">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="roaming"/>
+			<bridge-type-ref id="bluegiga"/>
+			<bridge-type-ref id="bluez"/>
+		</supported-bridge-type-refs>
+
+		<label>Grundfos Alpha Reader MI401</label>
+		<description>An Grundfos Alpha Reader MI401</description>
+		<category>pump</category>
+
+		<channels>
+			<channel id="rssi" typeId="rssi"/>
+			<channel id="flowRate" typeId="grundfos_flow"/>
+			<channel id="pumpHead" typeId="grundfos_head"/>
+			<channel id="pumpTemperature" typeId="grundfos_temperature"/>
```

Comment:
Please adapt according to [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention).
```suggestion
			<channel id="pump-temperature" typeId="grundfos-temperature"/>
```